### DISCUSSION TITLE
Restart on reboot

### DISCUSSION THREAD

**Discussion by almostheaven1863** [cryptosharks131/lndg#207] [General]
I have lndg installed on an Ubuntu 22.04 machine. My power went out and the service went down. I was able to restart the docker using docker-compose up -d in the lndg folder. Is there a way to set it to automatically restart upon reboot?

**Comment by scissorstail** [comment]
How about [this](https://docs.docker.com/config/containers/start-containers-automatically/)?

example) `docker update --restart always lndg`

Working?

**Reply by almostheaven1863** [reply]
it sounds like it would work, but I get an error. to run the docker, I go into my lndg directory and then run docker-compose up - d. 
Now it is currently running. While I am still in the directory I tried to run docker update --restart always lndg. This produced an error stating Error response from daemon: No such container: lndg

**Reply by scissorstail** [reply]
Run `docker ps` and find  the `NAMES` that are most likely to be lndg. and try again...

**Comment by almostheaven1863** [comment]
I ran that command and found a lndg-lndg-1. Then I used your other suggestion. After a reboot, the page still works. So thank you.
